src/Blockcore.Samples/Blockcore.Samples.csproj

```diff
@@ -2,15 +2,15 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
   </PropertyGroup>
 
    <ItemGroup>
       <PackageReference Include="Blockcore.Jose" Version="1.0.2" />
-      <PackageReference Include="Blockcore.NBitcoin" Version="1.0.12" />
-      <PackageReference Include="Blockcore.Networks.City" Version="1.0.12" />
-      <PackageReference Include="RestSharp" Version="106.11.4" />
-      <PackageReference Include="RestSharp.Serializers.NewtonsoftJson" Version="106.11.4" />
+      <PackageReference Include="Blockcore.NBitcoin" Version="1.0.14" />
```

Comment:
Remove and add the Blockcore.Core or something other that is where that code is now.